frontend/ui/widget/menu.lua

```diff
@@ -1315,6 +1299,21 @@ function Menu:onSelectByShortCut(_, keyevent)
     return true
 end
 
+function Menu:searchStringInMenuItems(search_string, goto_letter)
+    search_string = Utf8Proc.lowercase(util.fixUtf8(search_string, "?"))
+    for i, item in ipairs(self.item_table) do
+        if not item.is_go_up and not (goto_letter and item.is_file == false) then -- skip folders in "Go to letter"
+            local item_text = Utf8Proc.lowercase(util.fixUtf8(item.text, "?"))
+            local idx = item_text:find(search_string)
+            if idx and (idx == 1 or not goto_letter) then
+                self.itemnumber = i -- draw focus
+                self:onGotoPage(self:getPageNumber(i))
```

Comment:
I think this function is badly named: you don't search and return results (like what we do in other functions named with "Search"), you "go" to the first page containing or starting with the string.
`:goToMenuItemMatching(...)` may be?